src/libraries/System.Reflection/tests/TypeInfoTests.cs

```diff
@@ -733,6 +733,7 @@ public void GetConstructor(Type[] types, int? expected)
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/mono/mono/issues/15029", TestRuntimes.Mono)]
```

Comment:
https://github.com/mono/mono/pull/20535 seems to be closed when #44081 was merged. Should it be reopened if the bug has regressed since?

Reply:
Good catch! I think this PR (43407) is older than 44081 - i'll verify that the ActiveIssue is resolved and open a new PR to remove it

Reply:
Fix is in https://github.com/dotnet/runtime/pull/46680